src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -457,39 +457,6 @@ private set
         }
 
         /// <summary>
-        /// WARNING: This method is meant for unit test only. It directly accesses
-        /// the EngineController for the mirror data without waiting for any 
-        /// possible execution to complete (which, in single-threaded nature of 
-        /// unit test, is an okay thing to do). The right way to get the cached 
-        /// value for a NodeModel is by going through its RequestValueUpdateAsync
-        /// method).
-        /// </summary>
-        /// <param name="engine">Instance of EngineController from which the node
-        /// value is to be retrieved.</param>
-        /// <returns>Returns the MirrorData if the node's value is computed, or 
-        /// null otherwise.</returns>
-        /// 
-        internal MirrorData GetCachedValueFromEngine(EngineController engine)
-        {
-            if (cachedValue != null)
-                return cachedValue;
-
-            // Do not have an identifier for preview right now. For an example,
-            // this can be happening at the beginning of a code block node creation.
-            if (AstIdentifierForPreview.Value == null)
-                return null;
-
-            cachedValue = null;
-
-            var runtimeMirror = engine.GetMirror(AstIdentifierForPreview.Value);
-
-            if (runtimeMirror != null)
-                cachedValue = runtimeMirror.GetData();
-
-            return cachedValue;
-        }
-
-        /// <summary>
```

Comment:
Glad this is now gone, as the warning comment indicated this is a dangerous method. Wondering what's happening to the test cases that use it though... may have been removed.